Training24/intlist.c

```diff
@@ -0,0 +1,108 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------------------------------------
+// intlist.c
+// Program on A2 branch.
+// ------------------------------------------------------------------------------------------------
+
+#include <stdio.h>
+#include <malloc.h>
+#include "intlist.h"
+
+void Create (LinkList* list) {
+   list->head = NULL;
+   list->size = 0;
+}
+
+void DeleteList (LinkList* list) {
+   struct Node* current = list->head;  // Start from the head of the list

```

Comment:
Before you do anything, you should always check in all your functions if list* is NULL ie, the list exists